contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -99,8 +91,10 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
     }
 
     // Add recipient to the allow list
-    s_allowedSenders.add(recipient);
-    emit AddedAccess(recipient);
+    bool newAllowedSenderAdded = s_allowedSenders.add(recipient);
+    if (newAllowedSenderAdded) {
```

Comment:
Saves 13 gas to use avoid using a variable :slightly_smiling_face: 

```
if (s_allowedSenders.add(recipient)) {
```

Reply:
nice! will change that, thanks!